projects/backend/capi/articles.ts

```diff
@@ -84,18 +82,35 @@ const parseArticleResult = async (
     ]
 }
 
+const printsent = (paths: string[]) =>
+    `https://preview.content.guardianapis.com/content/print-sent?ids=${paths.join(
```

Comment:
This entire path should be considered a secret and will need to be passed in as config/ENV.

Reply:
I discussed this with @blishen prior to adding this to the PR.